apps/frontend/src/globals.scss

```
@@ -1,25 +1,54 @@
-@import url('https://fonts.googleapis.com/css2?family=Righteous&family=Poppins:wght@300;400;500;600;700&display=swap');
+/* Geist字体来自Vercel CDN */
+@font-face {
+    font-family: 'Geist';
```

Stylelint (17.7.0)

[error] Expected no quotes around "Geist" (font-family-name-quotes)

(font-family-name-quotes)